Client/Application/LeanplumIntegration.swift

```diff
@@ -217,6 +232,9 @@ class LeanplumIntegration {
     }
 
     func setUserAttributes(attributes: [AnyHashable : Any]) {
+        guard enabled else {
```

Comment:
same as above

Reply:
`shouldSendToLP` checks, but on the main thread. This is, perhaps, an unnecessary optimization.